### PR TITLE
edit namevaluelist example

### PR DESCRIPTION
**Pull request by britt6612**
[Deploy Preview](https://deploy-preview-INSERT_PR_#_HERE--keen-mayer-a86c8b.netlify.app/)

#### What does this PR do?
WIP for edit `namevaluelist` 
waiting for final designs 
#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:
- [ ] Console is free of warnings and errors
- [ ] Light & dark modes
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Keyboard interactions
- [ ] Screen reader experience

### CODE REVIEW

File: aries-site/src/pages/components/namevaluelist.mdx [hunk 1]
```diff
@@ -99,6 +100,21 @@ The pairProps direction determines how the name and value are aligned. If you wa
   <NameValueListPairPropsExample />
 </Example>
 
+### Name-Value Pair as a read-only display for Text Inputs
+In some cases, Name-Value Pairs may be used as read-only display for text inputs that are editable inline. Here, the name and value text styling should match the Text Input component (sans border).
```
Comment (taysea) [inline aries-site/src/pages/components/namevaluelist.mdx:104]:
```suggestion
In some cases, NameValuePairs may be used as read-only display for text inputs that are editable inline. Here, the name and value text styling should match the Text Input component (sans border). 
```

File: aries-site/src/pages/components/namevaluelist.mdx [hunk 2]
```diff
@@ -99,6 +100,21 @@ The pairProps direction determines how the name and value are aligned. If you wa
...
+### Name-Value Pair as a read-only display for Text Inputs
+In some cases, Name-Value Pairs may be used as read-only display for text inputs that are editable inline. Here, the name and value text styling should match the Text Input component (sans border). 
+
+<Example
+  docs="https://v2.grommet.io/namevaluelist?theme=hpe#layout"
+  code={["https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/NameValueListGridExample.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/data.js",
+  ]}
+  height={{ min: 'small' }}
+  pad="large"
```
Comment (taysea) [inline aries-site/src/pages/components/namevaluelist.mdx:112]: This is case where we needed to override `template` padding to have it uniform --  we can add this to our punch list when we audit.

**Review by taysea — APPROVED** [review]
LGTM -- nice work on this

File: aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditExample.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+      {!edit ? (
+        <NameValueList pairProps={{ direction: 'column' }}>
+          {Object.entries(currentData).map(([name, value]) => (
+            <NameValuePair
+              name={<Text {...theme.formField.label}>{name}</Text>}
+            >
+              {name === 'Email' ? (
+                <Anchor label={value} href={`mailto:${value}`} />
+              ) : (
+                <Text {...theme.global.input.font}>{value || '--'}</Text>
```
Comment (halocline) [inline aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditExample.js:50]: Design calls for an 'xxsmall' spacing between label and value.

Additionally, design calls for a 'medium' gap when in column. Seems like we need to adjust the theme to address?

**Review by halocline — COMMENTED** [review]
Looking good. Getting into micro-tuning.

File: aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditExample.js (hunk 3 above)
Comment (taysea) [reply aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditExample.js:50]: Thoughts?

Comment (halocline) [reply aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditExample.js:50]: That was my thinking on both as well.

Comment (britt6612) [reply aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditExample.js:50]: looking making these changes now will tag both of you once its up

Comment (britt6612) [reply aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditExample.js:50]: PR to fix the spacing can be found in these 2 PRs
https://github.com/grommet/grommet/pull/5788
https://github.com/grommet/grommet/pull/5789

**Pull request merged**